webapp-src/locales/en/translations.json

```diff
@@ -42,6 +42,7 @@
     "error-value-expected": "Value required",
     "grant-title": "{{- client}} requires access to the following scopes",
     "grant": "Grant access",
+    "grant-none-available": "No grants have been requested",
```

Comment:
I'd say `No requested scope is available` instead.
Unless I'm wrong from the beginning, I use the word 'scope' for the resource requested and the word 'grant' as the authorization a user has granted a client with one or more scopes. Do you agree?

Reply:
Hmm… I'm following you; my only concern with your proposal is it might give the impression of a server-side technical problem, as if the scope (or resource) as such did not exist. Is this indeed intended message? My understanding is rather that the resource may well exist but the user has no permissions to access it. I might be entirely wrong, though.

Reply:
How about `"You don't have access to the requested scopes"`?

Reply:
That sounds fine to me.